testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/security/ClientConfigProviderTestJarHelper.java

```diff
@@ -120,13 +122,39 @@ static Process runClientConfigProviderBearerTestJar(TestType testType, String ja
     }
 
     static Process runClientConfigProviderTestJar(String jarPath, String[] args) throws IOException {
-        // use quotation marks for classpath on windows because folder names can have spaces
-        String classPath = System.getProperty("os.name").contains("indows")
-                ? "\"" + jarPath + ";" + System.getProperty("java.class.path") + "\""
+        final Pattern pattern = Pattern.compile(
+                ".*(maven|plexus|sisu|xnio|jsoup|compress|shrinkwrap|IntelliJ|mockito|arquillian|aether|wildfly|glassfish|hibernate|netty|xerces|xalan|hamcrest|io7m|slf4j|objenesis|sshd|marshalling|remoting|bytebuddy).*");
+
+        final String cp = System.getProperty("os.name").contains("indows")
+                ? jarPath + ";" + System.getProperty("java.class.path")
                 : jarPath + ":" + System.getProperty("java.class.path");
-        return Runtime.getRuntime()
-                .exec("java -cp " + classPath + " " + ClientConfigProviderTestJarHelper.PACKAGE_NAME + "."
-                        + String.join(" ", args));
+        final String[] paths = cp.split(Pattern.quote(File.pathSeparator));
+        final StringBuilder newCp = new StringBuilder();
+        for (String s : paths) {
+            final Path path = Path.of(s);
+            if (pattern.matcher(path.toString()).matches()) {
+                continue;
+            }
+            newCp.append(path).append(File.pathSeparator);
+        }
+        newCp.deleteCharAt(newCp.length() - 1);
```

Comment:
Why are we doing this?

Reply:
Originally the above method left a trailing path separator in the new Class Path, after more testing I think it work even with the trailing separator so we can remove this